math.lua

```diff
@@ -146,4 +179,8 @@ function flib_math.lerp(num1, num2, amount)
   return num1 + (num2 - num1) * flib_math.clamp(amount, 0, 1)
 end
 
+--- The number of ticks in a real unit of time.
+--- @return number
+flib_math.time = { second = 60, minute = 3600, hour = 216000, day = 5184000 }
```

Comment:
I don't really like having these in `math`. Part of my plan is to create a `time` module and move `flib_misc.ticks_to_timestring` into it. These would be good values to have in there as well.